src/min-reviewers.ts

```diff
@@ -0,0 +1,101 @@
+import { Label, PullContext, Review } from "./types";
+import * as core from "@actions/core";
+import * as github from "@actions/github";
+
+export function getMinReviewers(labels: Label[]): string | number {
```

Comment:
This function will return either number or `all`. Since in this case `string` type is not strict, we could add a type that string can only be `all` and make this function return types stricter.